stream/stream_lavf.c

```diff
@@ -439,4 +439,3 @@ const stream_info_t stream_info_ffmpeg_unsafe = {
   .stream_origin = STREAM_ORIGIN_UNSAFE,
   .can_write = true,
 };
-
```

Comment:
stray change

Reply:
Change it to this? Still need Squash Commits?

Reply:
> Still need Squash Commits?

yes

Reply:
Completed Squash Commits